airflow/auth/managers/fab/fab_auth_manager.py

```diff
@@ -62,6 +70,24 @@ def get_cli_commands() -> list[CLICommand]:
             SYNC_PERM_COMMAND,  # not in a command group
         ]
 
+    def get_api_blueprint(self) -> None | Blueprint:
+        """Return a blueprint of the API endpoints proposed by this auth manager."""
+        folder = Path(__file__).parents[0].resolve()  # this is airflow/auth/managers/fab/
+        with folder.joinpath("openapi", "v1.yaml").open() as f:
+            specification = safe_load(f)
+        api = FlaskApi(
+            specification=specification,
+            resolver=_LazyResolver(),
+            base_path="/security/v1",
```

Comment:
I would rather have the endpoint `/auth/fab/v1`

Reply:
Why? The endpoints don’t seem to be fab-specific to me, no need to couple them to the name.

Reply:
They are. They are provided by the FAB auth manager. These endpoints wont be available if the environment is using another auth manager

Reply:
Also, we don't need a common path either if other auth managers decided to add similar endpoints. I like `/auth/fab/v1`.

Reply:
yeah having the `/auth/` prefix is good, it'll allow us to recommend other providers to use it as well for a more consistent experience (like `/auth/aws` for instance). I was not a big fan of leaking out the fact that the default auth manager is fab, but if there is an agreement I'll happily comply.